.github/workflows/build-doxygen.yml

```
@@ -44,7 +44,7 @@ jobs:
 
     - name: Assign Doxygen PR reviewer
       run: |
-        [[ $GITHUB_ACTOR == "KAClough" ]] && PR_REVIEWER="SamuelBrady"
+        PR_REVIEWER="KAClough" && [[ $GITHUB_ACTOR == "KAClough" ]] && PR_REVIEWER="SamuelBrady"
```

This looks like the code I wrote before but I think it might be wrong (though I am ill so could not be thinking straight). I think it should be something like
```suggestion
if [[ $GITHUB_ACTOR == "KAClough" ]]; then
  PR_REVIEWER="SamuelBrady"
else
  PR_REVIEWER="KAClough"
fi
```
(I was getting a bit confused with Bash logical operators so thought it might be easier to switch to `if... then... else`.)

---

I don't understand why this would be any different but I'll believe it anyway

---

Maybe I'm misunderstanding the order of the logical ANDs without parentheses...

---

No problem :) should be sorted now